app/services/school_search.rb

```diff
@@ -15,6 +19,10 @@ def initialize(query: nil, limit: DEFAULT_LIMIT, lead_schools_only: false)
   end
 
   def call
+    self
+  end
+
+  def specified_schools
```

Comment:
I'm not entirely sure if we want to break away from including the service pattern here.
Wondering if it would be worth returning an object instead of `self`.
This way `SchoolSearch` itself is in control of the data exposed, rather than relying on an controllers/components to poke around its properties.
```suggestion
  def call
    OpenStruct.new(schools: specified_schools, limit: limit)
  end

  def specified_schools
```

Reply:
Thanks, i've pushed up the changes!